conda/activate.py

```diff
@@ -641,7 +641,7 @@ def _build_activate_shell_custom(self, export_vars):
         # The signature of this method may change in the future.
         if on_win:
             import ctypes
-            export_vars.update({"PYTHONIOENCODING": ctypes.cdll.kernel32.GetACP(),})
+            export_vars.update({"PYTHONIOENCODING": ctypes.cdll.kernel32.GetACP(), })
```

Comment:
I didn't touch this file, however flake8 was complaining about this file in my PR, hence the change to this file.

Reply:
this line looks like it's been removed 👍 needs a merge conflict fix, though